languages/cpp/exercises/concept/numbers/numbers.h

```diff
@@ -0,0 +1,6 @@
+#ifndef NUMBERS_H
+#define NUMBERS_H
+
+namespace assembly_line {}
```

Comment:
The stub file does not contain stubs for the methods the student needs to implement. These should be added (see [the docs](https://github.com/exercism/v3/blob/master/docs/concept-exercises.md#stub-implementation-file)).

Reply:
Added